core/src/com/google/inject/internal/BytecodeGen.java

```diff
@@ -63,7 +63,7 @@
  */
 public final class BytecodeGen {
 
-  static final Logger logger = Logger.getLogger(BytecodeGen.class.getName());
+  private static final Logger logger = Logger.getLogger(BytecodeGen.class.getName());
```

Comment:
This `Logger` is used from within an (anonymous) inner class [here](https://github.com/yaziza/guice/blob/checkstyle/core/src/com/google/inject/internal/BytecodeGen.java#L137).  Making it private causes the field to be accessed indirectly through a synthetic method, which I'm assuming is why it was package-private in the first place.  Since it's not like it was `public` before, IMO this change actually reduces the code quality.